### PR TITLE
feat(Culture/Curriculum): Add components

### PR DESCRIPTION
**Pull request by nsbtr**
Starting to build the Culture page!

### CODE REVIEW

File: src/components/modules/Split.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  `};
+  ${mq.tablet`
+    padding: ${props => props.theme.paddingTablet};
+  `};
+  ${mq.mobile`
+    padding: ${props => props.theme.paddingMobile};
+  `};
+`;
+
+const Split = ({ id, src, content, ctas, secondary, isReversed }) => {
```
Comment (nsbtr) [inline src/components/modules/Split.js]: @johnathanfong Feel free to suggest a better name for this or any other component. Couldn't think of one 😅

Comment (johnathanfong) [reply src/components/modules/Split.js]: @nsbtr is this not just the `SplitHero` component?

File: src/components/Button.js [hunk 2]
```diff
@@ -2,17 +2,19 @@ import styled from "styled-components";
...
-    props.secondary ? props.theme.white : "none"};
+  background: ${props => props.theme.secondaryAccent};
+  border: none;
   color: ${props => props.theme.white};
   font-family: ${props => props.theme.secondaryFont};
   font-weight: ${props => props.theme.bold};
   font-size: ${props => props.theme.ctaFontSize};
   text-transform: ${props => props.theme.ctaCasing};
+
+  &.secondary {
```
Comment (johnathanfong) [inline src/components/Button.js]: we can have unlimited variable styling using classNames @nsbtr

File: src/components/Tout.js [hunk 3]
```diff
@@ -23,18 +23,15 @@ const Tout = ({ image, title, description, secondary, ctas }) => {
...
+      {ctas && ctas.length > 0 ? (
         <CtaContainer>
           {ctas.map((cta, n) => (
-            <Button
-              key={`${n}-cta`}
-              text={cta.title}
-              href={cta.href}
-              style={cta.style}
-            />
+            <Button key={`${n}-cta`} href={cta.href} className={`${cta.style}`}>
```
Comment (johnathanfong) [inline src/components/Tout.js]: assigns `className` based on our `"style": "value"` (json)

File: src/routes/Home.js [hunk 4]
```diff
@@ -1,22 +1,95 @@
...
+import ApplicationForm from 'components/ApplicationForm';
+import H1 from 'components/H1';
+import BodyTextLockup from 'components/BodyTextLockup';
+import mainImage from 'media/bali-landscape-1.jpg';
+import villaImage from 'media/villa-malaathina-1.jpg';
+import studentsImage from 'media/students.jpg';
+import reflection from 'media/temple-sunset.jpg';
 
 const Home = props => {
+  /// This is placeholder content being used while our resources/data files are sorted out
```
Comment (johnathanfong) [inline src/routes/Home.js]: @JesseNoseworthy

**Comment by johnathanfong** [comment]
Closing this PR as I've pulled the work from this branch into https://github.com/colindamelio/saraswati/pull/15 cc @nsbtr @JesseNoseworthy

**Pull request closed without merging**